jetty/jetty12/src/main/java/com/linecorp/armeria/server/jetty/AbstractJettyServiceBuilder.java

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright 2024 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.jetty;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.function.Consumer;
+
+import org.eclipse.jetty.server.Handler;
+import org.eclipse.jetty.server.Handler.Singleton;
+import org.eclipse.jetty.server.HttpConfiguration;
+import org.eclipse.jetty.server.RequestLog;
+import org.eclipse.jetty.server.Server;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
+import com.linecorp.armeria.common.annotation.Nullable;
+
+/**
+ * A skeletal builder implementation for {@link JettyServiceBuilder} in Jetty 9 and Jetty 10+ modules.
+ */
+public abstract class AbstractJettyServiceBuilder {
```

Comment:
Question) What do you think of moving the deprecated methods to `jetty11/JettyServiceBuilder` and sharing the implementation of `AbstractJettyServiceBuilder` between `jetty11` and `jetty12`?

Reply:
`JettyServiceBuilder` exists in both `jetty11` and `jetty9`. Let me move the deprecated methods to them.